README.md

```diff
@@ -341,8 +341,11 @@ The Moodle Gitpod template supports the following environment variables:
 
 * `MOODLE_REPOSITORY`. The Moodle repository to be cloned. The value should be URL encoded. If left undefined, the default repository `https://github.com/moodle/moodle.git` is used.
 * `MOODLE_BRANCH`. The Moodle branch to be cloned. If left undefined, the default branch `main` is employed.
+* `DATAFILE`. When specified, this feature file initializes the Moodle site with essential data. The value should be URL encoded.
```

Comment:
Maybe you should explicitly say that it must be a URL? When I read about it initially, it was not clear. Only because I saw the `wget` I realised that.

Related to this... is there any example of generator-scenario available anywhere? IMO it would be great to point people to some example and docs, so they know what they can do.

Reply:
Good suggestions! I've improved this information with the following content, pointing out to the official place where these behat generators are explained. I'll try to add a sample to the official devdocs page too:

> `DATAFILE`. When specified, this feature URL initializes the Moodle site with essential data. The value should be URL encoded. The content of this file adheres to the [Behat generators format](https://moodledev.io/general/development/tools/generator#create-a-testing-scenario-using-behat-generators) for creating testing scenarios.

Reply:
I've created a pull-request to add more information to the behat generators tool: https://github.com/moodle/devdocs/pull/1013